src/components/PrimaryPanes/Outline.js

```diff
@@ -68,19 +68,18 @@ export class Outline extends Component<Props> {
     if (!selectedSource) {
       return;
     }
-
-    const sourceLine = func.location.start.line;
-    const functionText = getFunctionText(sourceLine);
-
     const copyFunctionItem = {
       id: "node-menu-copy-function",
       label: copyFunctionLabel,
       accesskey: copyFunctionKey,
-      disabled: !functionText,
+      disabled: false,
```

Comment:
If you open the "original source" tab for a given pretty-printed tab, the menu item was disabled, so I don't know that we can hardcode `false` -- the Editor suffers from the same issue.

Reply:
If i got it right, current way to get function text is limited for some cases, for example minified sources. So we can't be sure that it'll work 100% even we have list of detected functions in Outline.
I suppose for now i just revert this change and create an issue to investigate what can be improved in `findFunctionText` helper?

Reply:
Yeah, let's go back to `!functionText` to be consistent.

Reply:
Done